sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -903,11 +903,13 @@ typedef struct CommandListBatchConfig {
   bool dynamic() const { return Size == 0; }
 } zeCommandListBatchConfig;
 
-static const zeCommandListBatchConfig ZeCommandListBatch = [] {
+static const zeCommandListBatchConfig ZeCommandListBatchConfig(bool IsCopy) {
   zeCommandListBatchConfig Config{}; // default initialize
 
   // Default value of 0. This specifies to use dynamic batch size adjustment.
-  const auto BatchSizeStr = std::getenv("SYCL_PI_LEVEL_ZERO_BATCH_SIZE");
+  const auto BatchSizeStr =
+      (IsCopy) ? std::getenv("SYCL_PI_LEVEL_ZERO_COPY_BATCH_SIZE")
```

Comment:
```suggestion
      IsCopy ? std::getenv("SYCL_PI_LEVEL_ZERO_COPY_BATCH_SIZE")
```
Curious that all your `c ? a : b` have parenthesis around `c` everywhere.

Reply:
Thanks for the correction. I changed it everywhere.